README.md

```diff
@@ -67,12 +106,12 @@ Also note that if you decide to run the API with a ***live*** cluster, you won't
 worker. This is because the live SNS topic cannot push messages to a local development machine, only to an endpoint 
 exposed to the internet. See the system achitecture here for more context: https://github.com/biomage-ltd/developer-docs/wiki/Biomage-Single-Cell-Platform:-Architecture
 
-### 3. Run the UI locally
+#### 3. Run the UI locally
```

Comment:
Can you call this `3. (Optional) Run the UI locally`? You don't need this to get the API working.